packages/web-app-admin-settings/src/components/Groups/SideBar/GroupInfoBox.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="oc-flex group-info oc-mb-l">
+    <avatar-image class="oc-mb-m" :width="80" :userid="_group.id" :user-name="_group.displayName" />
+    <span class="oc-text-muted group-info-display-name" v-text="_group.displayName"></span>
+    <span class="oc-text-muted" v-text="groupMembersText"></span>
+  </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent, PropType, unref } from 'vue'
+import { Group } from 'web-client/src/generated'
+import { useGettext } from 'vue3-gettext'
+export default defineComponent({
+  name: 'GroupInfoBox',
+  setup(props) {
+    const _group = computed(() => props.group as Group)
```

Comment:
I'd prefer to not use `as` whenever possible.

```suggestion
    const _group = computed<Group>(() => props.group)
```